src/containers/home/HomeContainer.tsx

```diff
@@ -21,6 +23,11 @@ const HomeContainer: React.FC = () => {
       <p className="mt-10">
         {data}
       </p>
+      <p className="mt-10">
+        {
```

Comment:
This can be { countries && countries.map((country) => (<div>{country.name}</div>)) }

Reply:
Updated